doc/doxygen/src/creating-an-application.md

```diff
@@ -0,0 +1,92 @@
+Creating an application                              {#creating-an-application}
+=======================
+
+[TOC]
+
+To create your own application you need to create a directory contating some 
+from the C file(s) containing your source code and a Makefile. A template
```

Comment:
"... some of ...", but I'd drop that and just write "you need to create a directory contating the C file(s) containing your source code and a Makefile"

Also "s/contaiting/containing".